bundle-plugin/pom.xml

```diff
@@ -21,12 +21,12 @@
         <dependency>
             <groupId>org.apache.maven</groupId>
             <artifactId>maven-plugin-api</artifactId>
-            <version>3.5.0</version>
+            <version>3.8.5</version>
```

Comment:
Any specific reason why this was upgraded?

Reply:
We were uncertain whether the observed behaviour was a bug in the Maven plugin api.